Amplify/AmplifyDevMenu/AmplifyDevMenu.swift

```diff
@@ -0,0 +1,42 @@
+//
+// Copyright 2018-2020 Amazon.com,
+// Inc. or its affiliates. All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+import UIKit
+import SwiftUI
+
+@available(iOS 13.0.0, *)
+public final class AmplifyDevMenu: TriggerDelegate {
+
+    weak var devMenuDelegate: DevMenuDelegate?
+    var longPressGestureRecognizer: LongPressGestureRecognizer?
+
+    init(delegate: DevMenuDelegate) {
+        self.devMenuDelegate = delegate
+        self.longPressGestureRecognizer = LongPressGestureRecognizer(devMenuDelegate: delegate, triggerDelegate: self)
+    }
+
+    public func onTrigger() {
+        showMenu()
+    }
+
+    private func showMenu() {
+        print("showMenu")
+        let viewController = UIHostingController(rootView: AmplifyDevMenuList())
+        devMenuDelegate?.presentationContext().rootViewController?
+            .present(viewController, animated: true, completion: nil)
+    }
```

Comment:
Guard the the rootViewController and show a warning message if there is no rootview controller.
```
guard let rootViewController = devMenuDelegate?.presentationContext().rootViewController else {
    // Print warning
    return
}
let viewController = UIHostingController(rootView: AmplifyDevMenuList())
rootViewController.present(viewController, animated: true)
```

Reply:
Also add #ifDebug macro